src/Packages/Passport/Runtime/Scripts/Private/PassportImpl.cs

```diff
@@ -113,7 +113,7 @@ public UniTask<bool> Login(bool useCachedSession = false, DirectLoginMethod dire
                 var task = new UniTaskCompletionSource<bool>();
                 _pkceCompletionSource = task;
                 _pkceLoginOnly = true;
-                _directLoginMethod = directLoginMethod;
+                _directLoginOptions = directLoginOptions ?? new DirectLoginOptions();
```

Comment:
Wouldn't this set the login method to `Email` if the caller doesn't pass in the `DirectLoginOptions`? (which would default  to email, but not have an email set?). 

Guess this would just open up the login page due to the missing email address when `LaunchAuthUrl` is called, but feels a bit awkward doing it this way 🤔